docker/hive/start.sh

```diff
@@ -12,7 +12,8 @@ mv /genesis/hive-genesis.json /hive-genesis.json && rm -fr /genesis
 
 # 2. Start Katana
 echo "Launching Katana..."
-RUST_LOG=warn katana --block-time 2000 --disable-fee --chain-id=0x$(jq -r '.config.chainId' hive-genesis.json) --genesis genesis.json &
+chain_id=$(printf '%x' $(jq -r '.config.chainId' hive-genesis.json))
```

Comment:
if chainid is 1337, then isn't this hex incorrect?

0x1337 != hex(int(1337))

Reply:
`printf '%x'` converts the decimale value to hex